luigi/target.py

```diff
@@ -23,8 +23,15 @@
 import io
 import os
 import random
+import shutil
 import tempfile
 import logging
+
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
```

Comment:
Just copy paste from hadoop.py (the only place StringIO is used at luigi). Should I remove it?